src/store/project.ts

```diff
@@ -0,0 +1,73 @@
+import Column from "../interfaces/Column";
```

Comment:
I figured to put action and reducer in a single file as it's not that big. `ACTIONS` is where we set constant strings for action functions. And eventually, the flow goes to `reducer`. Here is the flow of this file in a nutshell;
```
initialState -> ACTIONS -> [action functions(e.g. updated project, addedNewColumn, etc.)] -> reducer
```